### ISSUE TITLE
Clarify Accessibility Considerations section and refer to latest WCAG

### ISSUE THREAD

**Issue by matatk** [w3c/a11y-tracking#268]
## Proposed comment

Accessibility Considerations
https://w3c.github.io/screen-orientation/#accessibility-considerations

> As users can have their devices mounted in a fixed orientation (e.g. on the arm of a wheelchair), developers that expect users to rotate their device when <a href="#dfn-lock-the-screen-orientation" id="ref-for-dfn-lock-the-screen-orientation-6" class="internalDFN" data-link-type="dfn|abstract-op">locking the screen orientation</a> need to be aware of the <a href="https://www.w3.org/TR/WCAG21/" data-matched-text="[[[WCAG21]]]">Web Content Accessibility Guidelines (WCAG) 2.1</a>'s <a href="https://www.w3.org/TR/WCAG21/#orientation" id="ref-for-index-term-orientation-success-criterion-1">Orientation Success Criterion</a>. The criterion makes it <a href="https://www.w3.org/TR/WCAG21/#dfn-essential" id="ref-for-index-term-essential-1">essential</a> that content and functionality is available regardless of the <a href="#dfn-screen-orientations" id="ref-for-dfn-screen-orientations-6" class="internalDFN" data-link-type="dfn|abstract-op">screen orientation</a>. When a particular orientation is <a href="https://www.w3.org/TR/WCAG21/#dfn-essential" id="ref-for-index-term-essential-2">essential</a>, web applications must advise the user of the orientation requirements.

We have a couple of small suggestions to improve the clarity of this section.

* The WCAG term 'essential' is used twice.
  - The first occurrence of 'essential' is really meaning 'requires', so we suggest replacing it (and not having a link there).
  - The second occurrence could be clarified by adding the text 'for the web application to function'.
* There's a specref alias for 'the latest WCAG' you could use: it's just `wcag`.

For convenience, here's the text we're proposing (not reflecting the updated WCAG reference, but with URLs updated to point to WCAG 2.2):

> As users can have their devices mounted in a fixed orientation (e.g. on the arm of a wheelchair), developers that expect users to rotate their device when <a href="#dfn-lock-the-screen-orientation" id="ref-for-dfn-lock-the-screen-orientation-6" class="internalDFN" data-link-type="dfn|abstract-op">locking the screen orientation</a> need to be aware of the <a href="https://www.w3.org/TR/WCAG22/" data-matched-text="[[[WCAG21]]]">Web Content Accessibility Guidelines (WCAG) 2.2</a>'s <a href="https://www.w3.org/TR/WCAG22/#orientation" id="ref-for-index-term-orientation-success-criterion-1">Orientation Success Criterion</a>. The criterion requires that content and functionality is available regardless of the <a href="#dfn-screen-orientations" id="ref-for-dfn-screen-orientations-6" class="internalDFN" data-link-type="dfn|abstract-op">screen orientation</a>. When a particular orientation is <a href="https://www.w3.org/TR/WCAG22/#dfn-essential" id="ref-for-index-term-essential-2">essential</a> for the web pplication to function, the applications must advise the user of the orientation requirements.

## Instructions

This follows the process at https://www.w3.org/WAI/APA/wiki/Horizontal_Review#For_Specifications_2

1. Create the review comment you want to propose by replacing the prompts above these instructions, but **LEAVE ALL THE INSTRUCTIONS INTACT**.

2. Set a label to identify the spec: this starts with s: followed by the spec's short name. If you are unable to do that, ask a W3C staff contact to help.

3. Ask the APA WG to review your comment.

4. After discussion with the APA WG, raise an issue in the repository of the WG that owns the spec. Use the text above these instructions as the starting point for that comment, but add any suggestions that arose from the APA WG. In the other WG's repo, add an 'a11y-needs-resolution' label to the new issue.

5. Delete the text below that says 'url_for_the_issue_raised', then add in its place the URL for the issue you raised in the other WG's repository. Do NOT remove the initial '§ '. Do NOT use \[...](...) notation – you need to delete the placeholder, then paste the URL.

6. Remove the 'pending' label, and add a 'needs-resolution' tag to this tracker issue. 

7. Edit this issue to **REMOVE ALL THE INSTRUCTIONS & THE PROPOSED COMMENT**, ie. the line below that is '---' and all the text before it to the very start of the issue.

---

**This is a tracker issue.** Only discuss things here if they are a11y group internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ url_for_the_issue_raised
